docs/zh-cn/2.0.3/user_doc/guide/task/map-reduce.md

```diff
@@ -1,34 +1,71 @@
-# MapReduce(MR)节点
+# MapReduce(MR) 节点
 
-- 使用MR节点，可以直接执行MR程序。对于mr节点，worker会使用`hadoop jar`方式提交任务
+## 综述
 
+MR 任务类型，用于执行 MapReduce 程序。对于 MR 节点，worker 会通过使用 `hadoop jar` 的方式提交任务。
 
-> 拖动工具栏中的![PNG](https://analysys.github.io/easyscheduler_docs_cn/images/toolbar_MR.png)任务节点到画板中，如下图所示：
+## 创建任务
 
-## JAVA程序
+- 点击项目管理-项目名称-工作流定义，点击"创建工作流"按钮，进入 DAG 编辑页面。
+- 拖动工具栏中的 <img src="/img/tasks/icons/mr.png" width="15"/> 任务节点到画板中，如下图所示：
 
- <p align="center">
-   <img src="/img/mr_java.png" width="80%" />
- </p>
+## 任务参数
+
+- 节点名称：设置任务的名称。一个工作流定义中的节点名称是唯一的。
+- 运行标志：标识这个节点是否能正常调度,如果不需要执行，可以打开禁止执行开关。
+- 描述：描述该节点的功能。
+- 任务优先级：worker 线程数不足时，根据优先级从高到低依次执行，优先级一样时根据先进先出原则执行。
+- Worker 分组：任务分配给 worker 组的机器执行，选择Default，会随机选择一台 worker 机执行。
+- 环境名称：配置运行脚本的环境。
+- 失败重试次数：任务失败重新提交的次数。
+- 失败重试间隔：任务失败重新提交任务的时间间隔，以分为单位。
+- 延迟执行时间：任务延迟执行的时间，以分为单位。
+- 超时告警：勾选超时告警、超时失败，当任务超过"超时时长"后，会发送告警邮件并且任务执行失败。
+- 资源：是指脚本中需要调用的资源文件列表，资源中心-文件管理上传或创建的文件。
+- 自定义参数：是 MR 局部的用户自定义参数，会替换脚本中以 ${变量} 的内容。
+- 前置任务：选择当前任务的前置任务，会将被选择的前置任务设置为当前任务的上游。
+
+### JAVA/SCALA 程序
  
-- 主函数的class：是MR程序的入口Main Class的全路径
-- 程序类型：选择JAVA语言 
-- 主jar包：是MR的jar包
-- 命令行参数：是设置MR程序的输入参数，支持自定义参数变量的替换
-- 其他参数：支持 –D、-files、-libjars、-archives格式
+- 程序类型：选择 JAVA/SCALA 语言。
+- 主函数的 Class：是 MR 程序的入口 Main Class 的**全路径**。
+- 主程序包：执行 MapReduce 程序的 jar 包。
+- 任务名称：MR 任务名称（可以不填）。
+- 命令行参数：是设置 MR 程序的输入参数，支持自定义参数变量的替换。
+- 其他参数：支持 –D、-files、-libjars、-archives 格式。
 - 资源： 如果其他参数中引用了资源文件，需要在资源中选择指定
-- 自定义参数：是MR局部的用户自定义参数，会替换脚本中以${变量}的内容
+- 自定义参数：是 MR 局部的用户自定义参数，会替换脚本中以 ${变量} 的内容
 
-## Python程序
+### Python 程序
 
 <p align="center">
-   <img src="/img/mr_edit.png" width="80%" />
+   <img src="/img/mr_python.png" width="80%" />
```

Comment:
> Could you move this img to `/img/tasks/demo/` directory? and you should also change previous version doc. BTW, use markdown syntax `![img](link)` instead of html syntax

This picture is needed to delete, I will do it later.